server/src/server/Server.java

```diff
@@ -30,38 +30,30 @@
 public class Server implements AutoCloseable {
     private static final Logger LOG = LoggerFactory.getLogger(Server.class);
 
-    private final ServerID serverID;
     private final io.grpc.Server serverRPC;
 
-    private final KeyspaceManager keyspaceStore;
-
-    public Server(ServerID serverID, io.grpc.Server serverRPC, KeyspaceManager keyspaceStore) {
+    public Server(io.grpc.Server serverRPC) {
         // Lock provider
-        this.keyspaceStore = keyspaceStore;
         this.serverRPC = serverRPC;
-        this.serverID = serverID;
     }
 
     public void start() throws IOException {
-        initialiseSystemSchema();
         serverRPC.start();
     }
 
+    // NOTE: this method is used by Grakn KGMS and should be kept public
+    public void awaitTermination() throws InterruptedException {
+        serverRPC.awaitTermination();
+    }
```

Comment:
serverRPC is created by KGMS, so KGMS should be able to call `serverRPC.awaitTermination()`, I am probably missing something here, it just looks like an odd method

Reply:
As per our offline discussion, will keep this for now as it'll make more sense to solve it by re-architecting `Server` and `ServerFactory` as they have high overlap in terms of functionality.